src/index.html

```diff
@@ -12,6 +12,118 @@
   </head>
   <body>
     <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <!-- <script type="text/javascript" src="./main.js"></script> -->
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <div class="group-form-field">
+        <fieldset>
+          <legend>Personal information</legend>
+          <label class="form-field">
+            Surname:
+            <input type="text" autocomplete="off"
+            name="Surname" required>
+          </label>
+          <label class="form-field">
+            Name:
+            <input type="text" autocomplete="off"
+            name="name" required>
+          </label>
+          <label class="form-field">
+             How old are You?
+            <input name="old" type="number"
+              size="1" min="1"
+              max="100" value="12">
+          </label>
+          <label class="form-field">
+            Full date of birth:
+            <input type="data" name="FullDate"
+              required size="6"
+              placeholder="mm/dd/yyyy">
+          </label>
+          <label class="form-field">
+              I accept the term of the agreement
+            <input type="checkbox" 
+              name="iata" 
+              required>
```

Comment:
```suggestion
            <input 
              type="checkbox" 
              name="iata" 
              required
            >
```
Write all of the tags with 3+ attributes like this